common/util.go

```
@@ -633,13 +630,10 @@ func GetNodeUUID() (string, error) {
 }
 
 // isValidGUID returns true if the string is a valid guid in the 8-4-4-4-12 format.
-func IsValidUUID(guid string) (bool, error) {
-	valid, err := regexp.MatchString(
+func IsValidUUID(guid string) bool {
+	valid, _ := regexp.MatchString(
 		"^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$", guid)
-	if err != nil {
-		return false, err
-	}
-	return valid, nil
+	return valid
 }
```

for completeness, let's not ignore err

```
func IsValidUUID(guid string) bool {
    valid, err := regexp.MatchString(
        "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$", guid)
    // valid should not be true if there's an error.
    Assert((err == nil) || !valid)

    return (err != nil) && valid
}
```